### DISCUSSION TITLE
What's type of regex?

### DISCUSSION THREAD

**Discussion by kazuminn** [SigmaHQ/sigma#3658] [Q&A] [answered]
HI. sigma is grate. We are useing sigma to Hayabusa.
https://github.com/Yamato-Security/hayabusa

There is some regex engine.
then We want to select a suit regex engine like a full power.
Which Regex engine is the Sigma written for?

**Comment by frack113** [comment, marked as answer]
From https://github.com/SigmaHQ/sigma-specification/blob/main/Sigma_1_0_1.md#modifier-types
`re: value is handled as regular expression by backends. Currently, this is only supported by the Elasticsearch query string backend (es-qs). Further (like Splunk) are planned or have to be implemented by contributors with access to the target systems`

As the backend are in python for sigmac and Pysigma , python ?
Try a regex from a rule in https://regex101.com/r/MgB2Nt/1 , only .Net C# did not works.

**Reply by nasbench** [reply]
Also, take a look at the answer from Thomas in this issue https://github.com/SigmaHQ/sigma-specification/issues/25

**Reply by kazuminn** [reply]
Oh. I see.
thanks.

**Reply by nasbench** [reply]
No worries. If this answers your question. Could you please mark the issue as answered.

Thanks.
